test/runtests.jl

```diff
@@ -1,13 +1,14 @@
 using Test
 using Tenet
 using OMEinsum
-using Reactant
 using Adapt
 
 include("Utils.jl")
 
 const TENET_TEST_GROUP = lowercase(get(ENV, "TENET_TEST_GROUP", "all"))
 
+println("Running tests for group: $TENET_TEST_GROUP")
```

Comment:
if you use `@info` we can get nice colors in the logs

```suggestion
@info "Running tests for group: $TENET_TEST_GROUP"
```

Reply:
Sure! I think I will remove that, I am putting this now since I think this `ENV` is not properly set by `CI.yml`.